go/app/infra.go

```diff
@@ -12,6 +16,12 @@ var errImageNotFound = errors.New("image not found")
 type Item struct {
 	ID   int    `db:"id" json:"-"`
 	Name string `db:"name" json:"name"`
+	Category string `db:"category" json:"category"`
+	Image_Name string `db:"image_name" json:"image_name"`
```

Comment:
When writing go ,you should name in mixed cap(camel case)!: https://google.github.io/styleguide/go/guide#mixedcaps
```suggestion
	ImageName string `db:"image_name" json:"image_name"`
```